examples/scheduled-actions-workspace/vscode-settings-actions2/package.json

```diff
@@ -9,9 +9,10 @@
   },
   "main": "index.js",
   "dependencies": {
-    "json-fixer": "1.6.13"
+    "json-fixer": "1.6.12",
+    "lodash": "4.17.21"
   },
   "devDependencies": {
-    "lodash": "4.17.21"
+    "lodash": "^4.0.21"
```

Comment:
are the multiple versions of lodash supposed to be here?

Reply:
I have reverted this change